src/locale/locales/de/messages.po

```diff
@@ -1180,113 +1146,106 @@ msgstr "Neue Benachrichtigungen laden"
 
 #: src/view/com/feeds/FeedPage.tsx:189
 msgid "Load new posts"
-msgstr "Mehr Beiträge laden"
+msgstr "Neue Beiträge laden"
 
 #: src/view/com/composer/text-input/mobile/Autocomplete.tsx:95
 msgid "Loading..."
-msgstr "Laden..."
+msgstr "Wird geladen..."
 
 #: src/view/com/modals/ServerInput.tsx:50
 msgid "Local dev server"
-msgstr "Lokaler Entwicklungs-Server"
+msgstr "Lokaler Entwicklungsserver"
 
 #: src/view/screens/Moderation.tsx:136
 msgid "Logged-out visibility"
-msgstr "Ausgeloggte Sichtbarkeit"
+msgstr ""
 
 #: src/view/com/auth/login/ChooseAccountForm.tsx:133
 msgid "Login to account that is not listed"
 msgstr "Anmeldung bei einem Konto, das nicht aufgelistet ist"
 
-#: src/view/screens/ProfileFeed.tsx:472
-#~ msgid "Looks like this feed is only available to users with a Bluesky account. Please sign up or sign in to view this feed!"
-#~ msgstr "Es sieht so aus, als ob dieser Feed nur für Benutzer mit einem Bluesky-Konto verfügbar ist. Bitte melde dich an oder registriere dich, um diesen Feed zu sehen!"
-
 #: src/view/com/modals/LinkWarning.tsx:63
 msgid "Make sure this is where you intend to go!"
-msgstr "Vergewissern Sie sich, dass Sie dorthin gehen wollen!"
+msgstr "Vergewissere dich, dass du auch wirklich dorthin gehen willst!"
 
 #: src/view/screens/Profile.tsx:163
 msgid "Media"
 msgstr "Medien"
 
 #: src/view/com/threadgate/WhoCanReply.tsx:139
 msgid "mentioned users"
-msgstr "genannte Nutzer"
+msgstr "erwähnte Benutzer"
```

Comment:
"erwähnte Nutzer" or "erwähnte Personen" (should be consistent)

Reply:
` "erwähnte Personen"`